datalab/datalab_session/data_operations/median.py

```diff
@@ -1,5 +1,11 @@
+from io import BytesIO
```

Comment:
The python style guide has advice on how imports should be organized, e.g. standard lib, 3rd party, then local application imports https://peps.python.org/pep-0008/#imports